engine/src/main/java/org/destinationsol/game/input/Shooter.java

```diff
@@ -35,17 +35,17 @@
     Shooter() {
     }
 
-    public static float calcShootAngle(Vector2 gunPos, Vector2 gunSpeed, Vector2 ePos, Vector2 eSpeed, float projSpeed,
+    public static float calcShootAngle(Vector2 gunPos, Vector2 gunVelocity, Vector2 ePos, Vector2 eVelocity, float projSpeed,
```

Comment:
Come on, couldn't you as well rename the param/vars names to make sense? Like, what does the `e` before `Velocity` even mean? Applies to whole class.

Reply:
Well, didn't know when doing the refactor, but I will investigate where is it used if there is something more useful.